src/main/java/com/khutircraftubackend/auth/exception/UserNotFoundException.java

```diff
@@ -3,7 +3,7 @@
 public class UserNotFoundException extends RuntimeException {
     // Конструктор без параметрів
     public UserNotFoundException() {
-        super("User not found");
+        super("UserEntity not found");
     }
```

Comment:
why don't we make this constructor private and and force everyone use only parametrized constructors for this exception?